index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="dist/css/styles.css" />
+    <title>Pattern | Log in</title>
+  </head>
+
+  <body class="bg-primary">
+    <main class="mx-auto text-center" style="max-width: 500px">
+      <h1 class="mt-5">pattern</h1>
+
+      <!-- Log in form -->
+      <section class="mx-1 mt-5">
+        <form action="profile.html">
+          <div class="mb-2">
+            <label for="exampleInputEmail1" class="form-label"></label>
+            <input
+              type="email"
+              class="form-control rounded-pill"
+              id="exampleInputEmail1"
+              aria-describedby="emailHelp"
+              placeholder="example@email.com"
```

Comment:
HTML form validation is required by the CA guides. Even a simple `required` attribute works.